sdk/tvOS/OoyalaTVSkinSDK/Widgets/OOOoyalaTVOptionCell.h

```diff
@@ -0,0 +1,21 @@
+//
+//  OOOoyalaTVOptionCell.h
+//  OoyalaTVSkinSDK
+//
+//  Created by Ileana Padilla on 7/19/17.
+//  Copyright © 2017 ooyala. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface OOOoyalaTVOptionCell : UICollectionViewCell
+
+@property (strong, nonatomic) UILabel * optionLabel;
```

Comment:
for the code to look simpler, you can decide to remove both `strong` and `assign` attributes in properties since those are the default values if you don't put anything about it.